src/Dto/Filter/AmountSpentFilter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OneSignal\Dto\Filter;
+
+final class AmountSpentFilter extends AbstractFilter
+{
+    public const GT = '>';
+
+    public const LT = '<';
+
+    public const EQ = '=';
```

Comment:
As these are common, maybe better to add many of them to AbstractFilter?

Reply:
ok sure

Reply:
done